src/ci/docker/host-aarch64/aarch64-gnu-debug/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM ubuntu:22.04
+FROM ubuntu:25.04
```

Comment:
Question: doesn't this mean that we no longer test if we can build against Ubuntu 22?